### DISCUSSION TITLE
Ore Distribution

### DISCUSSION THREAD

**Discussion by PooPooSkin** [Povstalec/StargateJourney#76] [Q&A] [answered]
What is the ore distribution on Abydos?

**Comment by Povstalec** [comment, marked as answer]
Not sure about the actual precise numbers, but it should be more or less the same, though somewhat biased towards the Y=30 and above. And of course, Abydos Naqudah Spires just generally having the most Naquadah of all places (they're made of like 1% Naquadah, which doesn't sound like a lot, but given how large they are, it is a lot)
